### DISCUSSION TITLE
Projects Beta: Add "tracked in" column and allow sorting

### DISCUSSION THREAD

**Discussion by danm** [community/community#11272] [Projects and Issues]
I have a workflow where I create an issue (story/epic) and from there, create new issues using the `- [ ] ` task list.

Then I have a projects board that lists all issues but I would like to be able to group / sort them by the issue they are tracked by.

Additionally, to be able to add a column to the table view which shows the `tracked by` issue would be really useful, if not more useful than the grouping.

An issue acting as an "Epic" with issues assigned to them:
<img width="857" alt="Screenshot 2022-02-09 at 23 0141" src="https://user-images.githubusercontent.com/6394846/153305393-21139c3b-32d7-48f5-8c42-f5e0c0ea6143.png">

Example of an issue which is tracked by the above "epic"
<img width="853" alt="Screenshot 2022-02-09 at 23 01 12" src="https://user-images.githubusercontent.com/6394846/153305398-b1c58c1a-efbd-4ed9-8a3b-e0572c3b1f35.png">

**Comment by MarkBennett** [comment]
I'm with you on this. There are many times when it would be nice to be able to look at just the stories tracked in a particular issue so that I can be sure none have slipped through the cracks and aren't scheduled to be worked on.

**Comment by itcarroll** [comment]
Just a teensy bit of indentation (in Table views) would be an alternative to making a `tracked-in` column that is also group-able. Introducing filter keywords like `-is:tracked` and `tasks:>2` would customize visibility.

```
--
1  foo
--
2  bar
--
  1  bizz
--
    1  fee
--
    2  fi
--
  2  buzz
--
3  baz
--
  1  fee
--
  2  fum
```

```
filter = -is:tracked
--
1  foo
--
2  bar
--
3  baz
```

```
filter = tasks:>0
--
2  bar
--
  1  bizz
--
    1  fee
--
    2  fi
--
3  baz
```

The same issue could show up twice, as tracking currently allows. The `tracked-in` column proposed doesn't handle that case.
